Update info on how to run Dockerfile (#65)
lf-lang/lf-lang.github.io: packages/documentation/copy/en/reference/Containerized Execution.md

### Using docker build and docker run

> Suppose your LF source file is `Foo.lf`. When you run `lfc` or use the IDE to generate code, a file called `Foo.Dockerfile` will appear in the `src_gen/Foo` directory, see [Structure of an LF project](/docs/handbook/a-first-reactor#structure-of-an-lf-project) for more info. You can use this Dockerfile to build a Docker image as follows. First, navigate into the directory where `Foo.Dockerfile` is located. Then issue the command:

Reviewer: @erlingrj, I realize that this PR was created a long time ago. I think it's still relevant, but outdated, because I think the file we produce now is just called `Dockerfile`. Could you please again verify the correctness of the proposed text? Thanks!

Author: I just checked and this PR is still correct

Reviewer: Thanks!

Reviewer: ```suggestion
Suppose your LF source file is `src/Foo.lf`. When you run `lfc` or use the IDE to generate code, a file called `Dockerfile` and `docker-compose.yml` will appear in the `src_gen/Foo` directory, see [Structure of an LF project](/docs/handbook/a-first-reactor#structure-of-an-lf-project) for more info. You can now build Docker image as follows. First, navigate into the directory where `Dockerfile` is located. Then issue the command:
```

```

>    docker build -t foo -f Foo.Dockerfile .

Reviewer: ```suggestion
   docker build -t foo .
```
